src/Maker/MakeAuthenticator.php

```diff
@@ -84,6 +88,15 @@ public function interact(InputInterface $input, ConsoleStyle $io, Command $comma
         $manipulator = new YamlSourceManipulator($this->fileManager->getFileContents($path));
         $securityData = $manipulator->getData();
 
+        // Determine if we should use new security features introduced in Symfony 5.1
+        if (!empty($securityData['security']['enable_authenticator_manager']) && $securityData['security']['enable_authenticator_manager']) {
+            $this->useSecurity51 = true;
```

Comment:
We're only supporting 5.2, right? If so, let's make sure all the 51 stuff -> 52. And also, we would need some way to detect that the user is using 5.1 and yell at them that we only support 5.2.

Reply:
`class_exists(UserBadge::class)` can be used to detect 5.2 security

Reply:
done